composer.json

```diff
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "php": "^5.3.3|^7.0"
+        "php": "^5.3.3 || ^7.0"
```

Comment:
new syntax vs old syntax. See the last example: https://getcomposer.org/doc/04-schema.md#package-links